cid/common.py

```diff
@@ -162,11 +160,13 @@ def wrap(self, *args, **kwargs):
     def __loadPlugins(self) -> dict:
         if sys.version_info < (3, 8):
             from importlib_metadata import entry_points
```

Comment:
Seems like this should be moved to the top of the file

Reply:
yes. we can do that.

Reply:
Like that https://github.com/aws-samples/aws-cudos-framework-deployment/pull/383/commits/094a82a1b9d2954fd13193a89e91793048e931ac ?